values.yaml

```diff
@@ -132,6 +133,19 @@ postgresql:
     limits:
       memory: 4096Mi
 
+# Use this if you have external Postgresql database
+# in this case you should set postgresql.enabled to false above and enabled to true here
+# the postgresqlUsername role must have CREATEDB privilege
+# the postgresqlDatabase must exists and be owned by the postgresqlUsername role
+external_postgresql:
+  enabled: false
+  postgresqlHost: example.server
```

Comment:
comment out this block? and keep values only for variables which have default values defined in code.